eclipse-collections-code-generator/src/main/resources/impl/stack/mutable/primitiveArrayStack.stg

```diff
@@ -242,6 +242,12 @@ public class <name>ArrayStack extends Abstract<name>Stack
         }
         this.delegate = <name>ArrayList.newListWith(array);
     }
+
+    @Override
+    public Mutable<name>Stack toStack()
+    {
+        return <name>Stacks.mutable.withAll(this);
```

Comment:
This should use `<name>Stacks.mutable.withAllReversed(this)` to be consistent with the object stack.